core/src/main/java/org/polypheny/db/routing/RoutingPlan.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019-2021 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.db.routing;
+
+import java.util.List;
+import java.util.Map;
+import org.polypheny.db.util.Pair;
+
+
+/**
+ * Common interface for cached and non-cached routing plans.
+ */
+public interface RoutingPlan {
+
+    String getQueryClass();
+
+    String getPhysicalQueryClass();
```

Comment:
What is (or is there at all) a difference between physicalQueryId and physicalQueryClass?
While I am not happy with either of these terms, physicalQueryClass seems to be more appropriate.

Reply:
There is no difference between physicalQueryID and physicalQueryClass. But there is a difference between queryClass (only logical information of partitions) and physicalQueryClass (including physical information of partitions).